src/test/kotlin/com/terraformation/backend/tracking/model/PlantingZoneModelTest.kt

```diff
@@ -0,0 +1,176 @@
+    fun `does not choose permanent plots`() {
+      val model =
+          plantingZoneModel(
+              numTemporaryPlots = 1,
+              subzones = listOf(plantingSubzoneModel(plots = monitoringPlotModels(10, 11))))
+
+      repeatTest {
+        val chosenIds =
+            model.chooseTemporaryPlots(monitoringPlotIds(10)).map { it.id.value.toInt() }
+
+        assertEquals(listOf(11), chosenIds, "Should not have chosen permanent plot")
+      }
+    }
+
+    @Test
+    fun `spreads monitoring plots evenly across subzones`() {
+      val model =
+          plantingZoneModel(
+              numTemporaryPlots = 6,
+              subzones =
+                  listOf(
+                      plantingSubzoneModel(
+                          id = 1, plots = monitoringPlotModels(10, 11, 12, 13, 14)),
+                      plantingSubzoneModel(
+                          id = 2, plots = monitoringPlotModels(20, 21, 22, 23, 24)),
+                      plantingSubzoneModel(
+                          id = 3, plots = monitoringPlotModels(30, 31, 32, 33, 34))))
+
+      val availablePlotIds =
+          listOf(
+              setOf(11, 12, 13, 14),
+              setOf(21, 22, 23, 24),
+              setOf(30, 31, 32, 33, 34),
+          )
+
+      repeatTest {
+        val chosenIds =
+            model
+                .chooseTemporaryPlots(monitoringPlotIds(10, 20))
+                .map { it.id.value.toInt() }
+                .toSet()
+
+        val numChosenPerSubzone = availablePlotIds.map { ids -> ids.intersect(chosenIds).size }
+
+        assertEquals(listOf(2, 2, 2), numChosenPerSubzone, "Number of plots chosen in each subzone")
+      }
+    }
+
+    @Test
+    fun `places excess plots in subzones with fewest permanent plots`() {
+      val model =
+          plantingZoneModel(
+              numTemporaryPlots = 5,
+              subzones =
+                  listOf(
+                      plantingSubzoneModel(
+                          id = 1, plots = monitoringPlotModels(10, 11, 12, 13, 14)),
+                      plantingSubzoneModel(
+                          id = 2, plots = monitoringPlotModels(20, 21, 22, 23, 24)),
+                      plantingSubzoneModel(
+                          id = 3, plots = monitoringPlotModels(30, 31, 32, 33, 34))))
+
+      val availablePlotIds =
+          listOf(
+              setOf(11, 12, 13, 14),
+              setOf(21, 22, 23, 24),
+              setOf(30, 31, 32, 33, 34),
+          )
+
+      repeatTest {
+        val chosenIds =
+            model
+                .chooseTemporaryPlots(monitoringPlotIds(10, 11, 20))
+                .map { it.id.value.toInt() }
+                .toSet()
+
+        val numChosenPerSubzone = availablePlotIds.map { ids -> ids.intersect(chosenIds).size }
+
+        assertEquals(listOf(1, 2, 2), numChosenPerSubzone, "Number of plots chosen in each subzone")
+      }
+    }
+
+    @Test
+    fun `throws exception if temporary plot count not set`() {
```

Comment:
I probably missed this earlier, will setting 0 as the temporary plot count throw an exception?

Reply:
It's valid to not have any temporary monitoring plots, just permanent ones, though admittedly this is still a topic that's being discussed on the PRD.